target/linux/ath79/patches-4.14/410-add-OF-driver-for-mtd_concat-devices.patch

```diff
@@ -0,0 +1,226 @@
+From f7aa7d654326b21fce6d9904acb0d0c7b9e884d1 Mon Sep 17 00:00:00 2001
```

Comment:
I suggest put the two patches into target/linux/generic/backport-4.14 if they are accepted by upstream kernel.

Reply:
I have not submitted them upstream yet. This is my first kernel patch, I wanted to get some feedback first. Is this a strict requirement?

Reply:
Upstream will give you the feedback you are looking for.

Reply:
FYI, I have submitted the patches upstream: https://lkml.org/lkml/2018/9/8/135
I will now update this pull request and wait a little to see if the patch might get accepted (there have been concerns raised).